runtime/doc/lua.txt

```diff
@@ -2527,6 +2547,13 @@ vim.ui.input({opts}, {on_confirm})                            *vim.ui.input()*
                       typed (it might be an empty string if nothing was
                       entered), or `nil` if the user aborted the dialog.
 
+    >lua
+
+        vim.ui.input({ prompt = 'Enter value for shiftwidth: ' }, function(input)
+            vim.o.shiftwidth = tonumber(input)
+        end)
+<
```

Comment:
Example snippet got separated from `Example: ` text and placed after parameters??

Reply:
89838a0fa4ef2f6d7e8adb97e09d136f0fb9f940